public/docs/v2/components/paths/project/get-one.yaml

```
@@ -0,0 +1,242 @@
+get:
+  tags:
+    - Projetos
+  summary: Recupera um único projeto
+  description: Recuperar um único projeto
```

```suggestion
  description: Recuperar um único projeto
  parameters:
    - $ref: '../../parameters/path/id.yaml'
```
Acho que seria interessante deixar claro a necessidade de passar o identificador.